.coveragerc

```diff
@@ -1314,6 +1314,7 @@ omit =
     homeassistant/components/twitter/notify.py
     homeassistant/components/ubus/device_tracker.py
     homeassistant/components/ue_smart_radio/media_player.py
+    homeassistant/components/ukraine_alarm/*
```

Comment:
Please include the config flow in the coverage.

Reply:
Updated.